docs/pages/en-us/Features.md

```diff
@@ -31,9 +31,9 @@ The service is extensively scalable.  Elasticsearch is designed with scalability
 The service is designed for durability as Elasticsearch provides data redundancy across it's highly scalable cluster.
 
 ## High-Availability
-The service is highly available as Elasticsearch replicates and distributes data across it's scalable, durable cluster.  
+The service is highly available as Elasticsearch replicates and distributes data across it's scalable, durable cluster.
 
-## Audit Web Service API and Client
-Audit Web Service for logging application audit events through its RESTful interface which sends them through to Elasticsearch.
+## Audit Web Service
```

Comment:
Chris please run this by @glucy. I'm not sure Audit Web Service should the name of the key feature or whether it should be REST API Interface. The description should at least mention the use of a REST API interface to send audit events to Elasticsearch. Apart from this, I'm good with your changes.